client/my-sites/checkout/checkout-thank-you/google-apps-details.jsx

```diff
@@ -56,21 +57,27 @@ const GoogleAppsDetails = ( { purchases } ) => {
 		<PurchaseDetail
 			icon="mail"
 			title={ i18n.translate(
-				'Keep an eye on your email to finish setting up your G Suite account'
+				'Keep an eye on your email to finish setting up your %(productName)s account',
+				{
+					args: {
+						productName,
+					},
+				}
```

Comment:
It may be a good idea to add a comment here and in other translations to explain what is `productName`.